src/ProgressOnderwijsUtils/ProgressOnderwijsUtils.csproj

```diff
@@ -7,6 +7,7 @@
     <Description>Collection of utilities developed by ProgressOnderwijs</Description>
     <PackageTags>ProgressOnderwijs</PackageTags>
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks Condition="$(APPVEYOR)!='True'">netstandard2.0;netcoreapp3.0</TargetFrameworks>

```

Comment:
appveyor kan dit nog niet?

Reply:
Ik denk het wel, maar ik vermoed dat ik dan met een global.json aan de gang moet en daar heb ik niet zo'n zin in.

Reply:
ok!